src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKReadFilterPluginDescriptor.java

```diff
@@ -182,6 +187,9 @@ public boolean isDependentArgumentAllowed(final Class<?> dependentClass) {
             ReadFilter rf = readFilters.get(s);
             if (rf != null) {
                 filters.add(rf);
+            } else if (disableToolDefaultReadFilters) {
+                // include in the read instances since the tool one is disabled, but it is provided by the user
+                filters.add(toolDefaultReadFilters.get(s));
```

Comment:
In this PR the meaning of disable "all" the filters changed to disable only the defaults. I tried to solve the following problems by this change:

* What happen if `--disableDefaultReadFilters --readFilter DEFAULT_FILTER_NAME` is specified? I think that the user provided filters must be applied independently of the default set.
* How can an user specify always the same parameters independently of any change in the tool default filters? I think that one of the points of the new behavior for this parameter is to provide a solution to that question. Disabling default filters one by one is not possible if they change over time, because that requires changes in the command line too; thus, this is a shortcut to disabling all the default read filters without knowing them, and provide always the same set of filters to apply. 
* How can an user reorganize default filters to apply? The order of filtering may be important in some case, for instance if one of them is computationally expensive with respect to the others. As an example: there is only one tool default filter, which is expensive and the user wants to pre-filter by a less demanding one. Previously it was not possible, but with this change the command line will provide the following: `--disableToolDefaultReadFilters --readFilter CUSTOM --readFilter DEFAULT_EXPENSIVE`

I think that this proposal is worthy from the user point of view, even if the new contract is not yet applied. Either that or this change should go to the `getMergedReadFilter()`to allow the behavior explained above.

It is true that there is no test case for this, I've just implemented one.